<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonApply">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabAppearance">
    <attribute name="title">
     <string>Appearance</string>
    </attribute>
    <widget class="QLabel" name="labelVerticalList">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertical List:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonHorizontal">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonVertical">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>50</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Vertical</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSkinType">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>99</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Skin Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSkinType">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>211</x>
       <y>98</y>
       <width>200</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSkinDesign">
    <attribute name="title">
     <string>Skin Design</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetAllSkin">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelVerticalList_2">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>10</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>All Skin:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelVerticalList_3">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>136</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widgetSkinPreview" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>381</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>88</x>
       <y>115</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Description: Click to select and preview, double-click local to edit, save locally.</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
